packages/@react-spectrum/numberfield/docs/NumberField.mdx

```diff
@@ -0,0 +1,218 @@
+
+## Example
+
+```tsx example
+<NumberField label="Name" />
+```
+
+## Value
+
+A NumberField's `value` is empty by default, but an initial, uncontrolled, value can be provided using the `defaultValue` prop.
+Alternatively, a controlled value can be provided using the `value` prop.
+
+```tsx example
+function Example() {
+  let [value, setValue] = React.useState(15);
+
+  return (
+    <Flex gap="size-150" wrap>
+      <NumberField
+        label="Cookies (Uncontrolled)"
+        defaultValue={15} />
+
+      <NumberField
+        label="Cookies (Controlled)"
+        value={value}
+        onChange={setValue} />
+    </Flex>
+  );
+}
+```
+
+## Steps
+
+A NumberField can have a specified step size, as well as a min and max value. The steps will always start at the `minValue` or `0`
+and will not exceed the min or the max if they are specified.
+
+```tsx example
+<NumberField label="Cookies" step={3} minValue={-5} maxValue={21} />
+```
+
+## Labeling
+
+A visual label should be provided for the NumberField using the `label` prop. If the NumberField is required, the `isRequired` and
+`necessityIndicator` props can be used to show a required state.
+
+```tsx example
+<Flex gap="size-150" wrap>
+  <NumberField label="Cookies" />
+  <NumberField label="Cookies" isRequired necessityIndicator="icon" />
+  <NumberField label="Cookies" isRequired necessityIndicator="label" />
+  <NumberField label="Cookies" necessityIndicator="label" />
+</Flex>
+```
+
+### Accessibility
+
+If a visible label isn't specified, an `aria-label` must be provided to the NumberField for
+accessibility. If the field is labeled by a separate element, an `aria-labelledby` prop must be provided using
+the `id` of the labeling element instead.
+
+### Internationalization
+
+#### Label
+In order to internationalize a NumberField, a localized string should be passed to the `label` or `aria-label` prop.
+When the `necessityIndicator` prop is set to `"label"`, a localized string will be provided for `"(required)"` or `"(optional)"` automatically.
+
+#### Value
+The NumberField value can be formatted by using the `formatOptions` prop.
+`formatOptions` is compatible with the option parameter of [Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat) and is applied based on the current locale.
+
+```tsx example
+<Flex gap="size-150" wrap>
+  <NumberField
+    label="Currency"
+    formatOptions={{style: 'currency', currency: 'JPY'}}
+    defaultValue={60} />
+  <NumberField
+    label="Percent"
+    maxValue={1}
+    step={0.001}
+    formatOptions={{style: 'percent', minimumFractionDigits: 1}}
+    defaultValue={0.891} />
+</Flex>
+```
+
+NumberField also supports three numeral systems, Latin digits, Arabic-Indic digits, and Hanidecimal. Regardless of the locale, these three
+can be parsed from typed input. In addition, the NumberField will automatically choose the starting numeral system based on the default for the locale.
```

Comment:
> In addition, the NumberField will automatically choose the starting numeral system based on the default for the locale.

I don't quite follow this sentence. Or perhaps it's kinda redundant with the previous sentence where you said all three are supported?

Reply:
it was more to indicate how the initial display of a NumberField might look, for instance, if you're in the Arabic locale (in most cases) the field will render with the Arabic-Indic digits as opposed to Latin. After that, a user can clear it out and type Latin digits.